### PR TITLE
add 'fields' key when calling system/findXXX with describe

### PR DESCRIPTION
**Pull request by jdidion**
Add "fields" key to describe for all system/findXXX calls.

### CODE REVIEW

File: src/main/scala/dxWDL/dx/DxFindDataObjects.scala [hunk 1]
```diff
@@ -149,13 +149,19 @@ case class DxFindDataObjects(limit: Option[Int], verbose: Verbose) {
...
       withInputOutputSpec: Boolean
   ): (Map[DxDataObject, DxObjectDescribe], Option[JsValue]) = {
-    var fields = Set(Field.Name, Field.Folder, Field.Size, Field.ArchivalState, Field.Properties)
+    var fields = Set(Field.Name,
+                     Field.Folder,
+                     Field.Size,
+                     Field.ArchivalState,
+                     Field.Properties,
+                     Field.Created,
+                     Field.Modified)
```
Comment (commandlinegirl) [inline src/main/scala/dxWDL/dx/DxFindDataObjects.scala:158]: Why created and modified are needed? It doesn't look like we are using them anywhere (?)

File: src/main/scala/dxWDL/dx/DxJob.scala [hunk 2]
```diff
@@ -14,8 +14,7 @@ case class DxJobDescribe(project: String,
                          applet: DxApplet,
                          executableName: String,
                          parentJob: Option[DxJob],
-                         analysis: Option[DxAnalysis],
-                         output: Option[JsValue])
+                         analysis: Option[DxAnalysis])
```
Comment (commandlinegirl) [inline src/main/scala/dxWDL/dx/DxJob.scala:17]: Not sure why we don't need output from the job anymore.

File: src/main/scala/dxWDL/dx/DxFindDataObjects.scala (hunk 1 above)
Comment (jdidion) [reply src/main/scala/dxWDL/dx/DxFindDataObjects.scala:158]: They are extracted in the parseDescribe function

File: src/main/scala/dxWDL/dx/DxJob.scala (hunk 2 above)
Comment (jdidion) [reply src/main/scala/dxWDL/dx/DxJob.scala:17]: It is never used on the describe object. The only place the output field is used is in WorkflowOutputReorg, and it is parsed directly out of the JSON there.

**Review by commandlinegirl — APPROVED** [review]
LGTM, thank you!

**Pull request merged**